### ISSUE TITLE
Improve custom fields for the Development Board GitHub project

### ISSUE THREAD

**Issue by fjbo-net** [cotheca/docs#28]
## Description
In order to transition from a bi-project development cycle into a centralized Development Board, we need to add new meaningful custom fields.

### Objectives
Please specify what the finished feature should accomplish (one or more objectives):
- No code changes required. No feature branch required.
- [x] Add new custom field: **Status**
    - Field name: Status
    - Field type: Single Select
    - Options:
      - "New" (re-use existing one if available)
      This item hasn't been started.
      - "Blacklog" (re-use existing one if available)
      This item is actively being worked on
      - "Done" (re-use existing one if available)
      This item has been completed
- [x] Add new custom field: **Priority**
  - Field name: Priority
  - Field Type: Single select
  - Options:
    - Urgent (re-use existing one if available)
    - High (re-use existing one if available)
    - Medium (re-use existing one if available)
    - Low (re-use existing one if available)
- [x] Add new custom field: **Size**
  - Field name: Size
  - Field type: Single select
  - Options:
    - X-Large (re-use existing one if available)
    - Large (re-use existing one if available)
    - Medium (re-use existing one if available)
    - Small (re-use existing one if available)
    - Tiny (re-use existing one if available)
- [x] Add new custom field: **Category**
  - Field name: Category
  - Field type: Single select
  - Options:
    - Product/Project Management (re-use existing one if available)
    - Feature (re-use existing one if available)
    - Bug (re-use existing one if available)
    - Support (re-use existing one if available)
- [x] Add new custom field: **Target Date**
  - Field name: Target Date
  - Field type: Date
- [x] Add new custom field: **Start Date**
  - Field name: Start Date
  - Field type: Date
- [x] Add new custom field: **Sprint**
  - Field name: Sprint
  - Field type: Iteration
  - Iterations:
    - 2-week long
    - Start on closest Monday when the Cotheca Docs project was first created
    - Add past sprints up to the present day
    - Add at least the iterations a year quarter period for future (planned) sprints

## Additional Context
Current custom fields for the [Development Board](https://github.com/orgs/cotheca/projects/2)
![image](https://github.com/cotheca/docs/assets/28407576/f72a6748-61c5-4f2b-9056-4843f82689c8)

**Comment by fjbo-net** [comment]
Blocks:
- #27

**Comment by fjbo-net** [comment]
Blocks:
- #31

**Comment by fjbo-net** [comment]
Blocks:
- #32

**Comment by fjbo-net** [comment]
Custom field "Status" has been updated to issue objectives:

![image](https://github.com/cotheca/docs/assets/28407576/b45cb783-26e3-4bd4-981b-640d366a7fb0)

**Comment by fjbo-net** [comment]
Custom field "Priority" has been updated to fulfill issue's objectives:

![image](https://github.com/cotheca/docs/assets/28407576/ba68cea5-2036-4fd6-83b3-09e5aee417a3)

**Comment by fjbo-net** [comment]
Custom field "Size" is already up to issue's objectives:

![image](https://github.com/cotheca/docs/assets/28407576/e7c00bdb-6108-4a19-a9bf-4b2851843f9e)

Added an understandable definition for each size:

![image](https://github.com/cotheca/docs/assets/28407576/fb98db02-71ea-4831-a1df-8f1f11784504)

**Comment by fjbo-net** [comment]
Custom field "Category" is up to issue's objectives:

![image](https://github.com/cotheca/docs/assets/28407576/7ef940dc-0468-4153-a290-4333a518aa2a)

No changes needed.

**Comment by fjbo-net** [comment]
Added new custom field "Target Date":

![image](https://github.com/cotheca/docs/assets/28407576/80413bfe-d990-4524-8bec-023c8068e5ea)

**Comment by fjbo-net** [comment]
Added new custom field "Start Date":

![image](https://github.com/cotheca/docs/assets/28407576/19c45b17-80b5-49a0-94ff-67091fd22bc7)

**Comment by fjbo-net** [comment]
An iteration field cannot be set to the original creation of the Cotheca Project. Reviewing the previous sprints, I have concluded that naming does not overlap. Previously, Cotheca had 24 sprints in a year, and the sprint was approximately to 2-weeks. However, it was calendar-based starting the first sprint of any given month on the 1st of the month, and the 2nd sprint starting on the 16th of the month.

Going with a new numbering system starting on next Monday (04-29-2024) will make more sense as it's a new iteration system and does not reference the previous iteration system.

The new custom field "Sprint" has been created:

![image](https://github.com/cotheca/docs/assets/28407576/9f266fc1-f6b6-4fcb-a033-a0c787a5f698)

Five (planned) sprints have been added to cover 2024 Q2.
